src/basic/compress.c

```diff
@@ -237,7 +245,7 @@ int compress_blob_lz4(const void *src, uint64_t src_size,
         if (src_size < 9)
                 return -ENOBUFS;
 
-        r = sym_LZ4_compress_default(src, (char*)dst + 8, src_size, (int) dst_alloc_size - 8);
+        r = sym_LZ4_compress_HC(src, (char*)dst + 8, src_size, (int) dst_alloc_size - 8, level);
```

Comment:
Hm, is this change algorithm, especially picks slower one??

The documentation says level must be positive, so, please check the level.

Reply:
Yes, but should we pick any slower algorithm? I do not think so, at least for journal.

Reply:
what can you suggest?

Reply:
How about the following? Use `_default()` when level is neative or 0, otherwise use the slower algorithm with the specified level.